### PR TITLE
llnode 1.4.2

### PR DESCRIPTION
**Pull request by indutny**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Another update. Thanks!

### CODE REVIEW

File: Formula/llnode.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Llnode < Formula
   desc "LLDB plugin for live/post-mortem debugging of node.js apps"
-  homepage "https://github.com/nodejs/llnode"
-  url "https://github.com/nodejs/llnode/archive/v1.4.1.tar.gz"
-  sha256 "4c1e0143e0a580cb393b0c280f538427380a27bbaeb39a53b69f502955ca0231"
+  homepage "https://github.com/indutny/llnode"
```
Comment (dunn) [inline Formula/llnode.rb]: Leave this as `https://github.com/nodejs/llnode`, please!

Comment (indutny) [inline Formula/llnode.rb]: Oh yikes! Sorry, this was a leftover from invalid rebase. Fixed now!

File: Formula/llnode.rb [hunk 2]
```diff
@@ -20,7 +20,11 @@ class Llnode < Formula
   end
 
   resource "lldb" do
-    if MacOS::Xcode.version >= "8.0"
+    if MacOS::Xcode.version >= "8.3"
```
Comment (indutny) [inline Formula/llnode.rb]: Note: required lldb version changes with xcode updates, trying to keep the recent one here.

**Comment by ilovezfs** [comment]
@indutny thanks for the pull request!

**Comment by indutny** [comment]
Thank you!

**Pull request closed without merging**